src/components/o-testimonials/_preview.html

```diff
@@ -0,0 +1,24 @@
+<axa-testimonials classes="o-testimonials"
+                  title="Customer Reviews"
+                  auto-rotate-enabled="true"
+                  auto-rotate-time="5000"
+                  subtitle="AXA works hard to provide the best service possible to its customers.">
+  <div class="o-testimonials__item js-o-testimonial__item">
+    1111 Very helpful once I got through to the correct person but I was constantly being transferred from person to person,
+    in the end I called into my local axa once in the city and got sorted in 10 minutes after 3 weeks of phone calls.
+    <br/>
```

Comment:
better separate it with `<p>` tags and maybe consider the `<cite>` element (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/cite)

Reply:
I have these in 2 <p> tags in the local branch. <cite> seems to add italic to the content only and per design our content is not italic. 
Do you think I should add <cite> and override the italic part?

Reply:
`<cite>` fits semantically much better, so yes use it for sure.
Default styles can always be changed by CSS

Reply:
Done.